static/app/components/events/interfaces/llm-monitoring/llmMonitoringSection.tsx

```diff
@@ -37,17 +37,17 @@ export default function AIMonitoringSection({event, organization}: Props) {
       <LinkButton
         size="xs"
         icon={<IconOpen />}
-        to={`/organizations/${organization.slug}/ai-monitoring/`}
+        to={`/organizations/${organization.slug}/llm-monitoring/`}
```

Comment:
Can you use `useModuleURL` here? Otherwise this will get overlooked when I move the Insights URLs